backend/src/Service/Admin/Order/AdminOrderService.php

```diff
@@ -842,5 +850,25 @@ public function filterOrdersNotAnsweredByTheStore(FilterOrdersPaidOrNotPaidByAdm
 
         return $response;
     }
-    
+
+    public function checkWhetherCaptainReceivedOrderForSpecificStore(int $captainProfileId, int $storeId, int|null $primaryOrderId): int
+    {
+        $orderEntity = $this->adminOrderManager->checkWhetherCaptainReceivedOrderForSpecificStore($captainProfileId, $storeId);
+       
+        if (!empty($orderEntity)) {
```

Comment:
Instead of using empty with Entity object, we can just use the following condition: if ($orderEntity) { ... }